src/mantarray_desktop_app/constants.py

```diff
@@ -70,9 +70,9 @@
 VALID_CONFIG_SETTINGS = frozenset(
     [
         "customer_account_uuid",
-        "user_account_uuid",
+        "user_account_id",
         "customer_pass_key",
-        "customer_username",
+        "user_account_id",
```

Comment:
Can probably remove this second line since `user_account_id` is already in the set

Reply:
oh yes. my bad